Source/GitReleaseManager/AutoMapperConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+﻿using AutoMapper;
+
+namespace GitReleaseManager.Core
+{
+    public static class AutoMapperConfiguration
+    {
+        public static IMapper Configure()
+        {
+            var config = new MapperConfiguration(cfg =>
+            {
+                cfg.CreateMap<Model.Issue, Octokit.Issue>();
+                cfg.CreateMap<Model.Release, Octokit.Release>();
+                cfg.CreateMap<Model.Label, Octokit.Label>();
+                cfg.CreateMap<Model.Milestone, Octokit.Milestone>();
+                cfg.CreateMap<Octokit.Issue, Model.Issue>();
+                cfg.CreateMap<Octokit.Release, Model.Release>();
+                cfg.CreateMap<Octokit.Label, Model.Label>();
+                cfg.CreateMap<Octokit.Milestone, Model.Milestone>()
+                    .AfterMap((src, dest) => dest.Version = src.Version());

```

Comment:
The map configuration could be simplified by making use of `.ReverseMap()`.

Ending up with something like:
```cs
                cfg.CreateMap<Model.Issue, Octokit.Issue>().ReverseMap();
                cfg.CreateMap<Model.Release, Octokit.Release>().ReverseMap();
                cfg.CreateMap<Model.Label, Octokit.Label>().ReverseMap();
                cfg.CreateMap<Model.Milestone, Octokit.Milestone>();
                cfg.CreateMap<Octokit.Milestone, Model.Milestone>()
                    .AfterMap((src, dest) => dest.Version = src.Version());
```

See: https://docs.automapper.org/en/stable/Reverse-Mapping-and-Unflattening.html

Reply:
@AdmiringWorm did not know about this feature!  Thanks for sharing!